index.html

```diff
@@ -20,6 +20,7 @@
 <body>
   <div id="page-wrapper">
     <h1>My Personal Page!</h1>
+    <p1>I'm adding a git commit</p1>
```

Comment:
Headings have different levels: h1, h2, h3...
But paragraphs do not! They're just `<p>` tags!